test/lib/widgets/controller_node_test.rb

```diff
@@ -0,0 +1,105 @@
+#!/usr/bin/env rspec
+
+require_relative "../../test_helper"
+require "y2caasp/widgets/controller_node"
+
+describe Y2Caasp::Widgets::ControllerNode do
+  subject(:widget) do
+    Y2Caasp::Widgets::ControllerNode.new
+  end
+
+  let(:role) do
+    Installation::SystemRole.new(
+      id: "test_role", order: "100", label: "Test role", description: "Test description"
+    )
+  end
+
+  before do
+    allow(Installation::SystemRole).to receive(:current_role).and_return(role)
+    allow(Yast::Popup).to receive(:Error)
+  end
+
+  describe "#label" do
```

Comment:
hint: there is shared rspec examples for CWM. So something like
```
require "cwm/rspec"
...
includes_example "CWM::AbstractWidget"
```

for usage see e.g. https://github.com/yast/yast-storage-ng/blob/master/test/y2partitioner/widgets/btrfs_edit_button_test.rb#L17

Reply:
it is defined at https://github.com/yast/yast-yast2/blob/master/library/cwm/src/lib/cwm/rspec.rb

if you miss any, we can add it.